### DISCUSSION TITLE
Support for Geyserwise Max

### DISCUSSION THREAD

**Discussion by mac-za** [make-all/tuya-local#1000] [Q&A]
Good Afternoon

If you could please look at the provided info in order to create support for a Geyserwise Max control unit

I have the following information. Please let me know if I have omitted anything?

|DPID	|Description			|Example			|unit     |
|----------|--------------------------|---------------------|-------| 
|1		|Power				|On / Off			|-         |
|2		|Mode 				|Holiday or Normal	|-         |
|10		|Geyser Temp.		        |31				|Deg C |
|13		|Element				|On / Off			|-         |
|20		|Fault 				|E1 to E9			|-         |
|101		|Pump 				|On / Off			|-         |
|102		|Solar Differential	        |7				|Deg C |
|103		|Block 1				|55				|Deg C |
|104		|Block 2				|55				|Deg C |
|105		|Block 3				|55				|Deg C |
|106		|Block 4				|55				|Deg C |
|107		|Anti Freeze Temp.	        |5				|Deg C |
|108		|Collector			       |38				|Deg C |
|109		|Element Run Time	       |10				|hours  |

Thanks in advance

**Comment by make-all** [comment]
If you go to Issues and select "New Issues", and "New Device", then there are detailed instructions there for what information is required.

Leaving any info out from the template will result in your request being treated as low priority.  Reporting it as a discussion rather than an issue means it is not tracked, and will likely be buried under other discussions before it makes it to the front of the queue.

**Reply by mac-za** [reply]
Thank you

I will move it now
